client/app/components/packages/rwcds-form/with-action-no-action.hbs

```diff
@@ -0,0 +1,185 @@
+{{#let @form as |Form|}}
```

Comment:
`|Form|` here should be lower case `|form|` (in other PRs, we need to go through other templates and make this change, too. 

```
{{#let @form as |form|}}
```